tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -266,7 +298,7 @@ def get_logging_types():
     return logging_types
 
 
-def get_eks_config_basic(cluster_name):
+def get_eks_config_basic(cluster_name, launch_template=False):
```

Comment:
Did we start adding code for Launch templates also? Maybe we can have that in the next PR ? We can have this PR to just fix the existing test cases

Reply:
I have moved launch template code out of this PR.